src/Runner.Worker/Handlers/OutputManager.cs

```diff
@@ -84,7 +84,7 @@ public void OnDataReceived(object sender, ProcessDataReceivedEventArgs e)
             {
                 // This does not need to be inside of a critical section.
                 // The logging queues and command handlers are thread-safe.
-                if (_commandManager.TryProcessCommand(_executionContext, line))
+                if (_commandManager.TryProcessCommand(_executionContext, line, _container))
```

Comment:
OutputManager is the only place that calls commandManager.TryProcessCommand

In the OutputManager ctor, _container is set to \"step container ?? job container\". The change here is to pass the correct container for the command manager to use when translating paths.